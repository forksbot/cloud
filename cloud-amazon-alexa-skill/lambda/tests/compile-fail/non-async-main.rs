#[lambda]
#[tokio::main]
fn main() {}
