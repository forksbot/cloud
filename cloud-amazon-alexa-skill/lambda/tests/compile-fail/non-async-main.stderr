error: the async keyword is missing from the function declaration
 --> $DIR/non-async-main.rs:3:1
  |
3 | fn main() {}
  | ^^

error: cannot find attribute macro `lambda` in this scope
 --> $DIR/non-async-main.rs:1:3
  |
1 | #[lambda]
  |   ^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
  |
  = note: consider adding a `main` function to `$DIR/tests/compile-fail/non-async-main.rs`

For more information about this error, try `rustc --explain E0601`.
